dev/user.clj

```diff
@@ -242,12 +253,35 @@
 
 
   ;; Query Peer
-  (start {:fdb-query-peer-servers "localhost:8090"
-          :fdb-api-port            8080
-          :fdb-mode                "query"})
+  (start {:fdb-api-port 8080
+          :fdb-mode "query"
+          :fdb-query-peer-servers "localhost:8090"
+          :fdb-group-servers "query-server@localhost:11002"
+          :fdb-group-this-server "query-server"
+          :fdb-storage-type "memory"
+          :fdb-consensus-type "in-memory"})
```

Comment:
Can you align those k-v's please? (something something still need to add a .editorconfig one of these days haha)